src/index.js

```diff
@@ -3,19 +3,18 @@ import ReactDOM from 'react-dom';
 import App from './components/App';
 import { Provider } from 'react-redux';
 import { createStore, compose, applyMiddleware } from 'redux';
-import { createEpicMiddleware } from 'redux-observable'
+import { createEpicMiddleware } from 'redux-observable';
 import rootReducer from './reducers';
 import rootEpic from './epics';
-import ReduxDevTools from './components/_dev/ReduxDevTools';
 import 'bootstrap/dist/css/bootstrap.css';
 import './static/sass/main.scss';
 import './static/fonts/Lato.ttf';
 import './static/audio/notification_gertz.wav';
 
-const epicMiddleware = createEpicMiddleware(rootEpic)
+const epicMiddleware = createEpicMiddleware(rootEpic);
 const enhancer = compose(
   applyMiddleware(epicMiddleware),
-  ReduxDevTools.instrument()
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
@Spetastian We couldn't get `ReduxDevTools.instrument()` to work with the Redux Dev Tools Chrome extension.  Is there something special we need to do to be able to see Redux state in that extension while using `ReduxDevTools.instrument()`?

Reply:
@Spetastian Ping :-) . Curious how `ReduxDevTools.instrument()` is supposed to work and whether is does something more than `window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()` for us.

Reply:
Yeah, this is actually my misconception as to how setup using Redux devtools. The documentation was a bit vague. 

This here works as a basic setup and integration for using Redux devtools extension and it has a fallback if the extension is missing: https://github.com/cryptag/leapchat/pull/185